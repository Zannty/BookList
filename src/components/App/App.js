import React from "react";
import Header from "../Header/Header";
import Booklist from "../BookList/BookList";
const App = () => {
  return (
    <div className="app-wrap">
      <Header />
      <Booklist />
    </div>
  );
};

export default App;
