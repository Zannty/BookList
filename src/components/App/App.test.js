import React from "react";
import App from "./App";
describe("App", () => {
  const container = shallow(<App />);

  it("snap App", () => {
    expect(container.debug()).toMatchSnapshot();
  });
});
