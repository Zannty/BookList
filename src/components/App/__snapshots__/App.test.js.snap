// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App snap App 1`] = `
"<div className=\\"app-wrap\\">
  <Connect(Header) />
  <Connect(BookList) />
</div>"
`;
